readthedocs/doc_builder/environments.py

```diff
@@ -483,6 +483,10 @@ class BuildEnvironment(BaseEnvironment):
     :param version: Project version that is being built
     :param build: Build instance
     :param environment: shell environment variables
+    :param record: whether or not record a build commands in the databse via
+    the API. The only case where we want this to be `False` is when
+    instantiating this class from `sync_repository_task` because it's a
+    background task that does not expose commands to the user.
```

Comment:
Not certain, but I believe docutils field lists would want this indented. We don't render this anywhere of course, so not a big deal.

```suggestion
    :param record: whether or not record a build commands in the databse via
        the API. The only case where we want this to be `False` is when
        instantiating this class from `sync_repository_task` because it's a
        background task that does not expose commands to the user.
```